modules/apps/transfer/ibc_module.go

```diff
@@ -315,8 +322,8 @@ func (im IBCModule) OnChanUpgradeInit(ctx sdk.Context, portID, channelID string,
 		return "", err
 	}
 
-	if proposedVersion != types.Version {
-		return "", errorsmod.Wrapf(types.ErrInvalidVersion, "expected %s, got %s", types.Version, proposedVersion)
+	if !slices.Contains(types.SupportedVersions, proposedVersion) {
+		return "", errorsmod.Wrapf(types.ErrInvalidVersion, "invalid proposed version: expected one of [%s, %s], got %s", types.Version1, types.Version, proposedVersion)
```

Comment:
ditto here for error formatting, but should we be supporting upgrading a transfer channel where the transfer version stays as ics20-1? Is there a use-case where you'd want to enable ics29 fees for example but not upgrade to ics20-2? 

To me if I was going to upgrade the channel I'd do both I guess.

Reply:
not sure, but i dont think this method needs to be opinionated on whether or not ics29 needs to be used with ics20-2/channel upgrades to add mw need to also update ics20 version